### DISCUSSION TITLE
Theia Search Symbols Override

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#9934] [Q&A] [answered]
Hi All,
I wish to override existing theia's Search Symbol Quick Picker where I wish to type a few letters that would perform graphql calls with the typed keywords which would return the file icon followed by the file name. Any ideas on where to start would be of great help. 

Thanks

**Comment by vince-fugnitto** [comment, marked as answer]
@maheshrajrp are you referring to the `go to symbol...` command? If so then the behavior is contributed by the `monaco-editor` by default (you can verify by trying the [monaco playground](https://microsoft.github.io/monaco-editor/playground.html) and using <kbd>F1</kbd> in the editor). If its the behavior you are trying to customize, the best way may be to disable the command, and implement your own using the `quick-pick` service yourself.
